manifest_feature.yml

```diff
@@ -1,12 +1,16 @@
 ---
-inherit: manifest_base.yml
-host: fec-feature-proxy
+buildpack: staticfile_buildpack
+applications:
+  - name: proxy
+    memory: 64M
+routes:
+  - route: fec-feature-proxy.app.cloud.gov
 services:
   - fec-creds-feature
 env:
   PROXIES: |
     {
       "/": "https://fec-feature-cms.app.cloud.gov",
-      "/data": "https://fec-feature-web.app.cloud.gov",
       "/regulations": "https://fec-feature-eregs.app.cloud.gov"
     }
+  S3_BUCKET_URL: "https://cg-449d4df6-4b9e-4539-891f-363302ca5907.s3-us-gov-west-1.amazonaws.com"
```

Comment:
This is the separate S3 bucket for the `feature` space, correct?

Reply:
@ccostino I copied this over from the `manifest_base.yml` file.